lib/qldb.base.model.js

```diff
@@ -268,6 +268,8 @@ class Ledger {
      */
     async add(data, maxDepth=3)  {
         const modelCopy = cloneDeep(this.model);
+        //check if table exists or not. if not existed, creates table and insert data into it
+        await this.qldbConnection.checkTableExistence(this.tableName);
```

Comment:
This kills performance, would not do that here.

Reply:
Yes. I think soo.